### PR TITLE
8332671: Logging for pretouching thread stacks shows wrong memory range

### PR DESCRIPTION
**Pull request by tstuefe**
Trivial fix to a printout.

Before, looks like we only pretouch 32 bytes:

```
[2,417s][trace][os,thread] Pretouching thread stack from 0x00007f0f12640fe0 to 0x00007f0f12641000.          
```

Corrected with this patch:

```
[0,538s][trace][os,thread] Pretouching thread stack for 124618: [0x00007faa87f1afe0 - 0x00007faa87ffed90), (933296 bytes) (933296).
```

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issue
 * [JDK-8332671](https://bugs.openjdk.org/browse/JDK-8332671): Logging for pretouching thread stacks shows wrong memory range (**Bug** - P4)

### Reviewers
 * [Aleksey Shipilev](https://openjdk.org/census#shade) (@shipilev - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jdk.git pull/19342/head:pull/19342` \
`$ git checkout pull/19342`

Update a local copy of the PR: \
`$ git checkout pull/19342` \
`$ git pull https://git.openjdk.org/jdk.git pull/19342/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 19342`

View PR using the GUI difftool: \
`$ git pr show -t 19342`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jdk/pull/19342.diff">https://git.openjdk.org/jdk/pull/19342.diff</a>

</details>

### Webrev
[Link to Webrev Comment](https://git.openjdk.org/jdk/pull/19342#issuecomment-2124338136)

### CODE REVIEW

**Comment by tstuefe** [comment]
Windows test error unrelated.

**Comment by tstuefe** [comment]
None of the test errors are in any way relevant.

File: src/hotspot/share/runtime/javaThread.cpp [hunk 1]
```diff
@@ -2230,8 +2230,8 @@ void JavaThread::pretouch_stack() {
     if (is_in_full_stack(here) && here > end) {
       size_t to_alloc = here - end;
       char* p2 = (char*) alloca(to_alloc);
-      log_trace(os, thread)("Pretouching thread stack from " PTR_FORMAT " to " PTR_FORMAT ".",
-                            p2i(p2), p2i(end));
+      log_trace(os, thread)("Pretouching thread stack for " UINTX_FORMAT ": " RANGEFMT " (" SIZE_FORMAT ").",
+                            (uintx) osthread()->thread_id(), RANGEFMTARGS(p2, to_alloc), to_alloc);
```
Comment (shipilev) [inline src/hotspot/share/runtime/javaThread.cpp:2234]: Looks to me `RANGEFMTARGS` already prints the number of bytes in range, so printing `to_alloc` is excessive?

**Review by shipilev — APPROVED** [review]
Looks fine, with minor nits:

File: src/hotspot/share/runtime/javaThread.cpp (hunk 1 above)
Comment (tstuefe) [reply src/hotspot/share/runtime/javaThread.cpp:2234]: Grr. Shows that you can make an error in a trivial one line change. Thanks for catching.

**Review by shipilev — APPROVED** [review]

**Pull request closed without merging**